frontend/src/screens/Register/index.js

```diff
@@ -0,0 +1,136 @@
+                    hideProgressBar: false,
+                    closeOnClick: true,
+                    pauseOnHover: true,
+                    draggable: true,
+                    progress: undefined,
+                });
+
+            }
+            finally {
+                setFormData({ ...formData, name: '', email: '', password: '', confirmpassword: '' })
+            }
+
+        }
+        else {
+            toast.error(`Password does not match`, {
+                position: "top-right",
+                autoClose: 5000,
+                hideProgressBar: false,
+                closeOnClick: true,
+                pauseOnHover: true,
+                draggable: true,
+                progress: undefined,
+            });
+        }
+    }
+
+
+    return (
+        localStorage.getItem('TOKEN') ? (<Redirect to="/products" />) : (
+            <div className={styles.login__container}>
+                <h1>Register</h1>
+                <form onSubmit={onFormSubmit} className={styles.form}>
+                    <div className={styles.form_container}>
+                        <input
+                            className={styles.input_login}
+                            placeholder="Name"
+                            type="text"
+                            name="name"
+                            value={name}
+                            onChange={handleFormData}
+                        />
+                    </div>
+
+                    <div className={styles.form_container}>
+                        <input
+                            className={styles.input_login}
+                            placeholder="Email"
+                            type="email"
+                            name="email"
+                            value={email}
+                            onChange={handleFormData}
+                        />
+                    </div>
+
+                    <div className={styles.form_container}>
+                        <input
+                            className={styles.input_login}
+                            placeholder="Password"
+                            type="password"
+                            name="password"
+                            value={password}
+                            onChange={handleFormData}
+                        />
+                    </div>
+
+                    <div className={styles.form_container}>
+                        <input
+                            className={styles.input_login}
+                            placeholder="Confirm Password"
+                            type="password"
+                            name="confirmpassword"
+                            value={confirmpassword}
+                            onChange={handleFormData}
+                        />
+                    </div>
+                    <button className={`btn btn-primary ${styles.login_button}`}>Register</button>
+                    <ToastContainer
```

Comment:
You can add the `<ToastContainer>` in App.js, then you won't have to add to all files separately.

Reply:
Done ! Thanks for telling this. Now code looks more cleaner